### DISCUSSION TITLE
Sidebart doesn't show up in pages

### DISCUSSION THREAD

**Discussion by favinom** [mmistakes/minimal-mistakes#3920] [Q&A]
Hallo, why does not the sidebar appear in the pages?

Shall I set some options in _config.yml?

**Comment by iBug** [comment]
"Why" → You did something wrong.

Show us your repository where your problem can be reproduced, or we can help nothing.

See also: [How To Ask Questions The Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html)

**Reply by domsalisbury** [reply]
Looking at their respository, I think this problem is solved.

For others reading, you need 'author_profile:true' at the end of _config.yml (where it says Defaults), and you need to set this for the different types of scopes (e.g., pages and posts).
